compiler/rustc_mir_build/messages.ftl

```diff
@@ -312,6 +312,8 @@ mir_build_unreachable_pattern = unreachable pattern
     .label = unreachable pattern
     .catchall_label = matches any value
 
+mir_build_unsafe_not_inherited = items do not inherit unsafety from separate enclosing items
```

Comment:
Note that this is duplicated in `rustc_mir_transform`, along with all unsafety errors:

https://github.com/rust-lang/rust/blob/8bbeca4d9c1bff688abcafeaac5fe23e29eb6a35/compiler/rustc_mir_transform/messages.ftl#L43